### DISCUSSION TITLE
[M-Z] (2024/25) Esercizi per casa - Lezione 17/10/2024

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/algebra#532] [Esercizi M-Z]
1) Dimostrare che $2^n \not\equiv_n 1, \forall n>1$.
2) Mostrare che $\frac{\mathbb{Z}}{N\mathbb{Z}}$ è dominio se e solo se $N$ è primo (e $\frac{\mathbb{Z}}{N\mathbb{Z}}$ è un campo)
3) Dimostrare che dati $a,b,c\in N^*$,  $MCD(a,b,c)\mid MCD(a,c) \cdot MCD(b,c)$

**Comment by AglaiaNorza** [comment]
esercizi 2 e 3 ma non ci conterei tantissimo 

![image](https://github.com/user-attachments/assets/b6ff7e81-accb-42bf-8e9a-4a451e946308)
![image](https://github.com/user-attachments/assets/d1fe072e-500a-4c9c-b1e1-1d9e53fcb768)

**Comment by marcoseverini** [comment]
esercizio 1, interpretazione di una vecchia soluzione del prof 💀
![immagine](https://github.com/user-attachments/assets/3e679a83-507a-4b5f-9260-63b488d2f58d)
![immagine](https://github.com/user-attachments/assets/a8347a6f-a3d3-4bbb-92db-9c85cf511031)
![immagine](https://github.com/user-attachments/assets/5883009c-1a3c-4bf1-ae08-b5e7697f2df0)
